app/AvailablePlugin/DuplicateAccountEnroller/Model/DuplicateAccountEnroller.php

```diff
@@ -0,0 +1,171 @@
+ */
+
+class DuplicateAccountEnroller extends AppModel {
+  // Required by COmanage Plugins
+  public $cmPluginType = "enroller";
+
+  // Document foreign keys
+  public $cmPluginHasMany = array(
+    "CoEnrollmentFlowWedge" => "DuplicateAccountEnroller"
+  );
+
+  // Add behaviors
+  public $actsAs = array('Containable', 'Changelog' => array('priority' => 5));
+
+  // Association rules from this model to other models
+  public $belongsTo = array("CoEnrollmentFlowWedge");
+
+  public $hasMany = array();
+
+  // Default display field for cake generated views
+  public $displayField = "co_enrollment_flow_wedge_id";
+
+  // Validation rules for table elements
+  public $validate = array(
+    'co_enrollment_flow_wedge_id' => array(
+      'rule' => 'numeric',
+      'required' => true,
+      'allowEmpty' => false
+    ),
+    'env_remote_user' => array(
+      'rule' => '/.*/',
+      'required' => true,
+      'allowEmpty' => false
+    ),
+    'type' => array(
+      'content' => array(
+        'rule' => array('validateExtendedType',
+          array('attribute' => 'Identifier.type',
+                'default' => array(IdentifierEnum::AffiliateSOR,
+                                   IdentifierEnum::Badge,
+                                   IdentifierEnum::Enterprise,
+                                   IdentifierEnum::ePPN,
+                                   IdentifierEnum::ePTID,
+                                   IdentifierEnum::ePUID,
+                                   IdentifierEnum::GuestSOR,
+                                   IdentifierEnum::HRSOR,
+                                   IdentifierEnum::Mail,
+                                   IdentifierEnum::National,
+                                   IdentifierEnum::Network,
+                                   IdentifierEnum::OIDCsub,
+                                   IdentifierEnum::OpenID,
+                                   IdentifierEnum::ORCID,
+                                   IdentifierEnum::ProvisioningTarget,
+                                   IdentifierEnum::Reference,
+                                   IdentifierEnum::SamlPairwise,
+                                   IdentifierEnum::SamlSubject,
+                                   IdentifierEnum::SORID,
+                                   IdentifierEnum::StudentSOR,
+                                   IdentifierEnum::UID))),
+        'required' => true,
+        'allowEmpty' => false
+      )
+    ),
+    'redirect_url' => array(
+      'rule' => '/.*/',
+      'required' => false,
+      'allowEmpty' => true
+    )
+  );
+
+  /**
+   * Check for CO Person Identifier duplicates.
+   *
+   * @since  COmanage Registry v4.3.0
+   * @param  integer $coId                    CO ID
+   * @param  string  $env_remote_user         Identifier from Env
+   * @param  string  $ident_type              Identifier Type
+   * @return Array
+   */
+
+  public function findCoPersonDuplicate($coId, $env_remote_user, $ident_type) {
```

Comment:
findXXX prefixes can conflict with Cake's magic finding, these functions should probably have a different prefix (eg: `searchCoPersonDuplicates`).

Reply:
Done